tests/testthat/helper.R

```diff
@@ -29,8 +29,8 @@ scoped_temporary_thing <- function(dir = tempfile(),
 
   switch(
     thing,
-    package = capture_output(create_package(dir, rstudio = rstudio, open = FALSE)),
-    project = capture_output(create_project(dir, rstudio = rstudio, open = FALSE))
+    package = testthat::capture_output(create_package(dir, rstudio = rstudio, open = FALSE)),
+    project = testthat::capture_output(create_project(dir, rstudio = rstudio, open = FALSE))
```

Comment:
These aren't necessary and aren't used elsewhere in the tests. Can you remove?

In terms of test development, I load testthat in all interactive sessions in `.Rprofile` FWIW.

Reply:
Yes they are not necessary if testthat is loaded. I thought it might be good practice to not assume this is the case so I added them. But I will remove them  in that case. As it stands if now, the PR is anyways not intended to be merged, the primary goal was to make the unit tests passing, which was not possible.

Reply:
ok, removed.